docs/changelog.md

```diff
@@ -44,6 +44,9 @@ permalink: /changelog/
 * **lib-crash**
   * [Restrictions to background activity starts](https://developer.android.com/preview/privacy/background-activity-starts) in Android Q+ make it impossible to launch the crash reporter prompt after certain crashes. In those situations the library will show a "crash notification" instead. Clicking on the notification will launch the crash reporter prompt allowing the user to submit a crash report.
 
+* **support-ktx**
+  * Added `Context.isScreenReaderEnabled` extension to check if TalkBack service is enabled.
+
```

Comment:
Please move this up to `2.0.0-SNAPSHOT` :)

Reply:
:heavy_check_mark: